### PR TITLE
Assignment 8 pull request

### PR DESCRIPTION
**Pull request by cdelapena**
For future classes, please break-apart and expand on the instruction for lesson 8. This assignment was a far jump more difficult than the others, and I spent quite a bit of time spinning my wheels.

Please provide me with solution files after my assignment is graded, so that I can review the areas that I missed.

### CODE REVIEW

**Comment by cdelapena** [comment]
Closing pull request to create new one for assignment 9.

**Comment by cdelapena** [comment]
Closing pull request to create new one for assignment 9.

File: js/adopt-a-dog-blog.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    },
+    {
+        name: 'text',
+        label: 'Blog Text'
+    }
+];
+
+const form = document.createElement('form');
+
+document.body.appendChild(form);
```
Comment (MadEste) [inline js/adopt-a-dog-blog.js:18]: The above was not required for this assignment, but I will not take off points for it.

File: js/adopt-a-dog-blog.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+imageContainer.className = "grid-item-3";
+imageContainer.appendChild(image);
+
+//append image and text into 2x1 grid blogContainer
+//var container = document.createElement('div');
+var container = document.createElement('div');
+container.className ="grid-container";
+container.appendChild(imageContainer);
+container.appendChild(textContainer);
+
```
Comment (MadEste) [inline js/adopt-a-dog-blog.js:79]: Everything loads well!

File: adoptadog-blog.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      <a class="nav-link" href="adoptadog-dogs.html">Dogs / </a>
+      <a class="nav-link" href="adoptadog-blog.html">Blog / </a>
+      <a class="nav-link" href="adoptadog-checkout.html"> Checkout</a>
+    </nav>
+
+    <h1 style="padding:1em 10%;">Adoptadog Blog</h1>
+
+    <script src="./js/adopt-a-dog-blog.js"></script>
+
+    <div class="grid-container" id = "blogContainer1">
```
Comment (MadEste) [inline adoptadog-blog.html:38]: You Should not have elements hardcoded into the html, but generate all the html using JS.

**Pull request closed without merging**